### DISCUSSION TITLE
Cannot GET /login

### DISCUSSION THREAD

**Discussion by famousfive1** [NITK-KODE/cms-api#22] [General]
When I run the project using `npm run devc` or `dev` or just `npm start` and go to localhost the page just shows "Cannot GET /login". Is this supposed to happen? Is this a problem on my side or do I need to open an issue?

**Comment by priyansh32** [comment]
`/login` is a POST route, this is an api so it won't render any web page. You can use Postman to send POST requests or use Thunder Client extension on VS Code.
I would recommend using Postman, it has so many other features apart from just sending HTTP requests.

**Reply by famousfive1** [reply]
Oh. i thought it was supposed to GET a login page or something

**Reply by priyansh32** [reply]
We are planning to build the frontend with React.

**Reply by Utkar5hM** [reply]
You aren't supposed to get a login page via the backend but the current GET function at '/' works only when logged in which gives out information of currently loggedin User. 

The authentication middleware ('IsLoggedIn') Currently redirects you to /login when you try to request/fetch at the addresses 
 where the middleware is used, thats why you are getting redirected to it.

**Reply by BenzeneAlcohol** [reply]
Additionally, we are building only the APIs in the backend [REST API]. So nothing will be rendered on the backend, ie., there won't be any views folder. The frontend is completely separate and will be built using react.
